wp/lib/bap_wp/src/environment.ml

```diff
@@ -367,6 +373,10 @@ let get_sub_handler (env : t) (tid : Tid.t) : fun_spec_type option =
 let get_jmp_handler (env : t) : jmp_spec =
   env.jmp_handler
 
+(*ADDED*)
```

Comment:
Sorry for this tiny nitpick, but can we remove this comment?

Reply:
Sorry. I thoght I had all the comments and (*ADDED*)'s removed. I'll get rid of all of those.

Reply:
👍